README.md

```diff
@@ -351,6 +350,8 @@ the client used to send the request.
 | `client.insecure`                      | Whether to skip verifying the server's certificate chain and host name.     | `false`         |
 | `client.ignore-redirect`               | Whether to ignore redirects (true) or follow them (false, default).         | `false`         |
 | `client.timeout`                       | Duration before timing out.                                                 | `10s`           |
+| `client.df`                            | Option to set the DoNotFragement bit for ICMP packet                        | `false`         |
+| `client.size`                          | Size of the ICMP packet                                                     | `25`            |
```

Comment:
I'In hindsight, maybe this should be under `icmp`, i.e. 

```
client:
  icmp:
    df: true
    size: 25
```

What do you think?

Reply:
It's done, as you said, it makes more sense that way